debian/libjxl-gdk-pixbuf.install

```diff
@@ -0,0 +1,3 @@
+usr/lib/x86_64-linux-gnu/*/*/loaders/*
```

Comment:
This wouldn't work for other archs, right?

Reply:
Very true. Fixed.